hokusai/production.yml

```diff
@@ -30,12 +30,20 @@ spec:
       containers:
         - name: convection-web
           env:
+            - name: PORT
+              value: "8080"
+            - name: PUMA_WORKERS
+              value: "1"
```

Comment:
same as [default](https://github.com/artsy/convection/blob/fb3f7b6be672363c93c0ad2d81a4201275cca16c/config/puma.config#L10) that is in effect.